bin/cli.js

```diff
@@ -3,17 +3,17 @@
 'use strict';
 
 const minimist = require('minimist');
-const _ = require('lodash');
+const camelCase = require('lodash.camelcase');
 const checkDependencies = require('../lib/check-dependencies');
 
 const argv = minimist(process.argv.slice(2));
 
 // camelCase the options
-for (const key in argv) {
+Object.keys(argv).forEach(key => {
```

Comment:
`for in` is dangerous: it is slow and behaves differently in older V8 and newest V8 (will be shipped in Node 7)/other browsers.